### PR TITLE
v1.0.0

### PR DESCRIPTION
**Pull request by iAnanich**
- [x] Refactored RemoteEphyrInstance
- [x] Subscription API
- [x] Sidechain toggle for MixinOutput
- [x] update README

TODO before merge:
- [ ] run WPS style checks
- [ ] check for missing examples (like usage of State)
- [x] update changelist

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,15 @@ Ephyr Control changelog
 All user visible changes to this project will be documented in this file. This project uses [Semantic Versioning 2.0.0].
 
 
-## [vNext]
+## v1.0.0
```
Comment (skhalymon) [inline CHANGELOG.md:7]: Update with new changes

File: README.md [hunk 2]
```diff
@@ -6,14 +6,14 @@ Ephyr Control
 Client library that allow to control [Ephyr] streaming.
 
 ## Overview
-**Ephyr Control** allow to connect and control process of mixing of audio. Currently only changing audio volume is available. 
+**Ephyr Control** allow to connect and control process of mixing of audio.
```
Comment (skhalymon) [inline README.md:9]: "allow to connect and control behavior of Ephyr instance"

Comment (skhalymon) [reply README.md:9]: Update Readme

File: ephyr_control/__init__.py [hunk 3]
```diff
@@ -1,4 +1,10 @@
-__version__ = "0.1.0"
+"""Ephyr-control library.
 
-from .config import *
-from .control import *
+Set of tools for managing Ephyr instances.
+"""
+
+__version__ = "1.0.0"
```
Comment (skhalymon) [inline ephyr_control/__init__.py:6]: update version

Comment (skhalymon) [reply ephyr_control/__init__.py:6]: or add bumversion to project

File: ephyr_control/instance/constants.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+__all__ = (
+    "MIXIN_UI_PATH",
+    "EphyrPasswordKind",
+    "EphyrApiPaths",
+    "ALL_API_PATHS",
+)
+
+
+MIXIN_UI_PATH = "/mix"
```
Comment (skhalymon) [inline ephyr_control/instance/constants.py:11]: Add comment: uses for generating mixi urls

File: ephyr_control/instance/protocols.py [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+            raise KeyError(f"There is no client matching {method_call}")
+
+        return client.execute(
+            method_call=method_call,
+            variable_values=variable_values,
+        )
+
+
+class RemoteEphyrInstanceProtocol(EphyrInstanceProtocol, Protocol):
+    """EPhyr isntance hosted on server.
```
Comment (skhalymon) [inline ephyr_control/instance/protocols.py:150]: typo: EPhyr to Ephyr

**Review by skhalymon — DISMISSED** [review]
Approved with comments to update

File: ephyr_control/__init__.py (hunk 3 above)
Comment (iAnanich) [reply ephyr_control/__init__.py:6]: version is correct here

File: README.md (hunk 2 above)
Comment (iAnanich) [reply README.md]: Done

**Review by skhalymon — APPROVED** [review]

**Pull request merged**